docs/services-developer-guide.md

```diff
@@ -117,12 +117,17 @@ The first step toward implementing a new managed service is to define the code
 level schema of its configuration resources. These are referred to as
 [resources], (resource) [kinds], and [objects] interchangeably. The kubebuilder
 scaffolding is a good starting point for any new Crossplane API kind, whether
-they'll be a managed resource, resource class, or resource claim:
+they'll be a managed resource, resource class, or resource claim. Additionally the [Crossplane CLI] helps you to bootstrap a new stack:
+
+```console
+go mod init stacks.crossplane.io/favourite-cloud
```

Comment:
I usually see the module initialized with a remote repo location such as `github.com/mycoolorg/mycoolrepo`. Have you had a different experience?